src/node.cc

```diff
@@ -172,6 +172,9 @@ using v8::Value;
 static Mutex process_mutex;
 static Mutex environ_mutex;
 
+bool upstream_node_mode = true;
+bool standalone_mode = true;
```

Comment:
I'd actually imagine `standalone_mode` is used when Node.js is run standalone (not being embedded)..`standalone_mode` and `embedded_mode` sounds more intuitive to me.